### DISCUSSION TITLE
Very Helpful Video For HW 1 Part 3

### DISCUSSION THREAD

**Discussion by Sam-Besch** [cooperrc/computational-mechanics#149] [General]
Hey just thought I'd throw this out there, Professor Cooper's youtube video titled "Numerical integration (exponential growth)" is very helpful in completing part 3 of hw 1. the link to it is below 

https://www.youtube.com/watch?v=sFZsJnjVJkQ&list=PLhlHDqdX_NQ99dA6plaq0mQgWF4Fp4IXl&index=76

**Comment by cooperrc** [comment]
Thanks for curating!!
